src/Ironclad/Application/UserStore.cs

```diff
@@ -298,26 +335,46 @@ public Task<IList<ApplicationUser>> GetUsersForClaimAsync(Claim claim, Cancellat
 
         public Task AddLoginAsync(ApplicationUser user, UserLoginInfo login, CancellationToken cancellationToken)
         {
-            return Task.CompletedTask;
+            return ((IUserLoginStore<ApplicationUser>)this.userStoreOrig.Instance).AddLoginAsync(user, login, cancellationToken);
```

Comment:
Should not that call be awaited?

Reply:
no, we returning the task and the caller awaits it